diff -aurN libbinio-1.4.orig/doc/libbinio.info libbinio-1.4/doc/libbinio.info
--- libbinio-1.4.orig/doc/libbinio.info	2004-10-01 20:00:04.000000000 +0200
+++ libbinio-1.4/doc/libbinio.info	2006-02-09 22:22:44.924076792 +0100
@@ -18,9 +18,9 @@
      modify this GNU Manual, like GNU software.  Copies published by
      the Free Software Foundation raise funds for GNU development."
    
-INFO-DIR-SECTION Software Libraries
+INFO-DIR-SECTION Libraries
 START-INFO-DIR-ENTRY
-* libbinio: (libbinio)          Binary I/O stream class library 1.4
+* libbinio: (libbinio)		Binary I/O stream class library 1.4
 END-INFO-DIR-ENTRY
 
 
diff -aurN libbinio-1.4.orig/doc/libbinio.info-1 libbinio-1.4/doc/libbinio.info-1
--- libbinio-1.4.orig/doc/libbinio.info-1	2004-10-01 20:00:04.000000000 +0200
+++ libbinio-1.4/doc/libbinio.info-1	2006-02-09 22:23:00.032779920 +0100
@@ -18,9 +18,9 @@
      modify this GNU Manual, like GNU software.  Copies published by
      the Free Software Foundation raise funds for GNU development."
    
-INFO-DIR-SECTION Software Libraries
+INFO-DIR-SECTION Libraries
 START-INFO-DIR-ENTRY
-* libbinio: (libbinio)          Binary I/O stream class library 1.4
+* libbinio: (libbinio)		Binary I/O stream class library 1.4
 END-INFO-DIR-ENTRY
 
 
diff -aurN libbinio-1.4.orig/doc/libbinio.info-2 libbinio-1.4/doc/libbinio.info-2
--- libbinio-1.4.orig/doc/libbinio.info-2	2004-10-01 20:00:04.000000000 +0200
+++ libbinio-1.4/doc/libbinio.info-2	2006-02-09 22:23:19.892760744 +0100
@@ -18,9 +18,9 @@
      modify this GNU Manual, like GNU software.  Copies published by
      the Free Software Foundation raise funds for GNU development."
    
-INFO-DIR-SECTION Software Libraries
+INFO-DIR-SECTION Libraries
 START-INFO-DIR-ENTRY
-* libbinio: (libbinio)          Binary I/O stream class library 1.4
+* libbinio: (libbinio)		Binary I/O stream class library 1.4
 END-INFO-DIR-ENTRY
 
 
diff -aurN libbinio-1.4.orig/doc/libbinio.texi libbinio-1.4/doc/libbinio.texi
--- libbinio-1.4.orig/doc/libbinio.texi	2004-08-18 21:49:11.000000000 +0200
+++ libbinio-1.4/doc/libbinio.texi	2006-02-09 22:24:19.032770096 +0100
@@ -25,9 +25,9 @@
 @end quotation
 @end copying
 
-@dircategory Software Libraries
+@dircategory Libraries
 @direntry
-* libbinio: (libbinio)          Binary I/O stream class library @value{VERSION}
+* libbinio: (libbinio)		Binary I/O stream class library @value{VERSION}
 @end direntry
 
 @titlepage
